runironic.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/bash
+
+set -x
+
+# Optionally configure ironic if the environment variable is set.
+if [ ! -e "$CONFIGURE_IRONIC" ] ; then
```

Comment:
Hopefully we can eventually not need this and use the same config for both the install and BMO Ironic.  Since https://github.com/metalkube/metalkube-ironic/pull/14 sets the port to 5280 without a conditional this would cause a problem if not configuring Ironic here.

Reply:
I agree, I think we should configure ironic the same way in dev-scripts and on k8s. Meaning, the dev-scripts part should be removed.